vue.config.js

```diff
@@ -12,7 +12,6 @@ module.exports = {
     output: {
       filename: "js/[name]" + fileVersion + ".js",
       chunkFilename: "[name]" + fileVersion + ".js",
-      library: "@example/header",
```

Comment:
setting the output.library requires using the systemjs named-register extra, which is undesireable.